src/store/types.ts

```diff
@@ -80,7 +81,7 @@ export interface UnitScore {
 }
 
 export interface Hero {
-  icon: string;
+  name: string;
```

Comment:
You're identifying heros by their name in the frontend anyways?

Reply:
For the one I know yeah, I wasn't sure if other things use the existing `icon` field and didn't want to break them.

Reply:
Shouldn't we rather be using IDs?

Reply:
I would also say this struct will need to have some way to discriminate maps - if we go with ids, there could be multiple heroes with id H001, if we go with icon paths, multiple different icons could be Texture/Hero.dds

Reply:
The name is used for translation keys and paths to images, which make sense to be names to me. I changed to `name` instead of `icon` as I thought it's clearer, as it's not just for the icon.

Either way, there's no reason why `name` can't include a prefix of some kind if we want later which can be used across both asset path and translation keys.

Reply:
The backend has been updated to also have a `HeroSource` field for discriminating heroes